packages/python/accern_data/util.py

```diff
@@ -39,8 +42,19 @@ def is_test() -> int:
     return IS_TEST
 
 
+def check_filters(
+        record: Union[pd.DataFrame, Dict[str, Any]],
+        filters: 'FiltersType') -> bool:
+    for key, value in filters.items():
+        if record[key] != value:
```

Comment:
this won't work with df

Reply:
well the record type is lying

Reply:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/39483475/165350118-da7a321d-5d21-49dd-94c3-bbbaf915f7dc.png">

Reply:
don't go one by one